apps/mobile/src/components/ContextSwitchMenu/ContextSwitchMenu.js

```diff
@@ -0,0 +1,137 @@
+
+  return (
+    <Animated.View className='flex-col h-full bg-theme-background z-50 items-center py-2 px-3'>
+      {/* FlatList used instead of FlashList because of strict-sizing requirements of FlashList */}
+      <FlatList
+        data={myGroups}
+        keyExtractor={(item) => item.id}
+        renderItem={({ item }) => (
+          <NavRow
+            changeToGroup={changeToGroup}
+            item={item}
+            currentGroupSlug={currentGroup?.slug}
+          />
+        )}
+        showsVerticalScrollIndicator={false}
+      />
+      <View className='w-full mt-auto bg-theme-background pt-4'>
+        <TouchableOpacity 
+          onPress={() => {}} // TODO redesign: Needs to open some creation dialog...
+          style={styles.rowTouchable}
+          activeOpacity={0.7}
+        >
+          <View
+            className={`
+                bg-primary relative flex flex-col text-primary-foreground items-center justify-center
+                w-14 h-14 min-h-10 rounded-lg drop-shadow-md opacity-60 scale-90
+            `}
+          >
+            <Plus />
+          </View>
+        </TouchableOpacity>
+        <TouchableOpacity
+          onPress={() => {}} // Needs to open some creation dialog?
+          style={styles.rowTouchable}
+          activeOpacity={0.7}
+        >
+          <View className='bg-primary relative flex flex-col text-primary-foreground items-center justify-center w-14 h-14 min-h-10 rounded-lg drop-shadow-md opacity-60 scale-90'>
+            <CircleHelp />
+          </View>
+        </TouchableOpacity>
+      </View>
+    </Animated.View>
+  )
+}
+
+function NavRow ({ item, changeToGroup, currentGroupSlug, badgeCount = 0, className }) {
+  const { id, avatarUrl, name, slug } = item
+  const newPostCount = Math.min(99, item.newPostCount)
+  const highlight = slug === currentGroupSlug
+
+  return (
+    <TouchableOpacity
+      key={id} 
+      onPress={() => changeToGroup(item?.slug, false)}
+      style={styles.rowTouchable}
+      activeOpacity={0.7}
+    >
+      <View
+        className={[
+          'bg-primary relative flex flex-col items-center justify-center w-14 h-14 min-h-10 rounded-lg drop-shadow-md opacity-60 scale-90',
+          highlight && 'border-3 border-secondary opacity-100 scale-100',
+          badgeCount > 0 && 'border-3 border-accent opacity-100 scale-100',
+          className
+        ].filter(Boolean).join(' ')}
+      >
+        {!!avatarUrl && (
+          <FastImage source={{ uri: avatarUrl }} style={styles.groupAvatar} />
+        )}
+        {slug === PUBLIC_GROUP.slug &&
+          <View className='flex items-center w-14 h-14 min-h-10 rounded-lg'>
+            <Globe />
+          </View>
+        }
+        {!!newPostCount && (
+          <View style={styles.badge}>
+            <Text style={styles.badgeText}>{newPostCount}</Text>
+          </View>
+        )}
+      </View>
+    </TouchableOpacity>
+  )
+}
+
+const styles = StyleSheet.create({
```

Comment:
Let's talk at some point about what we keep do with NativeWind classNames and what we keep as React Native styles. I don't have the answer at the moment, and am not expecting us to know perfectly yet, but you may have a better sense than I already.

Reply:
At a minimum:
- Because of the themeing setup, we more or less have to port everything impacting color over to nativewind.
Beyond that, I don't have a strong impulse of a need either way.